cpu/stm32/stmclk/stmclk_l4wx.c

```diff
@@ -487,17 +487,16 @@ void stmclk_init_sysclk(void)
              ~100 kHz and the MSI clock to be based on MSISRANGE in RCC_CSR
              (instead of MSIRANGE in the RCC_CR) */
     RCC->CR = (RCC_CR_HSION);
-
+    /* Use VDDTCXO regulator, required by Radio */
+#if defined(CPU_FAM_STM32WL)
```

Comment:
if this is only needed for the radio, please use the following snippet:
```suggestion
#if defined(MODULE_SX126X_STM32WL)
```

Reply:
As discussed I have updated it to if `(IS_ACTIVE(CLOCK_ENABLE_HSE) || IS_USED(MODULE_SX126X_PERIPH)) {)`